go.mod

```diff
@@ -23,7 +24,7 @@ require (
 	github.com/antham/chyle v1.6.0
 	github.com/aws/aws-sdk-go v1.24.0
 	github.com/awslabs/goformation v0.0.0-20190320125420-ac0a17860cf1
-	github.com/banzaicloud/bank-vaults v0.0.0-00010101000000-000000000000
+	github.com/banzaicloud/bank-vaults v0.0.0-20190508130850-5673d28c46bd
```

Comment:
uhhh, I have bad recollections on the Vault version switching around

Reply:
Yeah, this makes that stop. =)